recipes/conversational_voice_agent/convert_and_optimize_asr.py

```diff
@@ -16,25 +142,45 @@ def convert_asr_model(model_dir: Path) -> Path:
     Returns:
        Path to exported model dir
     """
-    output_dir = model_dir / (MODEL_NAME.rsplit ("/")[-1] + "-FP16")
 
     # load model and convert it to OpenVINO
-    model = OVModelForSpeechSeq2Seq.from_pretrained(MODEL_NAME, export=True, compile=False)
-    # change precision to FP16
-    model.half()
-    # save model to disk
-    model.save_pretrained(output_dir)
+    output_dir = model_dir / (MODEL_NAME.rsplit ("/")[-1] + "-FP16")
+    ov_config = {"CACHE_DIR": ""}
+
+    if not output_dir.exists():
+        ov_model = OVModelForSpeechSeq2Seq.from_pretrained(
+            MODEL_NAME, ov_config=ov_config, export=True, compile=False, load_in_8bit=False
+        )
+        ov_model.half()
+        ov_model.save_pretrained(output_dir)
+    else:
+        ov_model = OVModelForSpeechSeq2Seq.from_pretrained(
+            output_dir, ov_config=ov_config, compile=False
+        )
 
-    # export also processor
     asr_processor = AutoProcessor.from_pretrained(MODEL_NAME)
+
+
+    if use_quantization: 
+        # Use post-training quantization of NNCF to quantize the ASR model
+
+        CALIBRATION_DATASET_SIZE = 50
+        quantized_distil_model_path = model_dir / (MODEL_NAME.rsplit ("/")[-1] + "-INT8")
+        ov_model.to("AUTO")
```

Comment:
Why AUTO device here? Shouldn't be CPU or nothing?
Is compilation needed?

Reply:
Hi Adrian, replaced codes with updated version, using Optimum-Intel for weights compression directly. Please help review. Thanks~